proteus/MeshTools.py

```diff
@@ -6001,3 +6001,75 @@ def getMeshIntersections(mesh, toPolyhedron, endpoints):
                 continue
             intersections.update(((tuple(elementIntersections[0]), tuple(elementIntersections[1])),),)
     return intersections
+
+
+from proteus import default_n as dn
+from proteus import default_p as dp
+
+class MeshOptions:
+    """
+    Mesh options for the domain
+    """
+    def __init__(self, domain=None):
+        self.Domain = domain
+        self.he = 1.
+        self.use_gmsh = False
```

Comment:
I agree with having the gmsh switch here in the MeshOptions (that's an unchecked item in your todo list, but I think you're done on that).